modules/ecs/main.tf

```diff
@@ -1,15 +1,42 @@
 module "network" {
   source               = "../network"
+
   environment          = "${var.environment}"
   vpc_cidr             = "${var.vpc_cidr}"
   public_subnet_cidrs  = "${var.public_subnet_cidrs}"
   private_subnet_cidrs = "${var.private_subnet_cidrs}"
-  availibility_zones   = "${var.availibility_zones}"
+  availability_zones   = "${var.availability_zones}"
+  owner                = "${var.owner}"
+  cluster              = "${var.cluster}"
   depends_id           = ""
 }
 
+module "rds" {
+  source             = "../rds"
+
+  cluster_name       = "${var.cluster}"
+  environment        = "${var.environment}"
+  vpc_id             = "${module.network.vpc_id}"
+  ecs_instance_sg_id = "${module.ecs_instances.ecs_instance_security_group_id}"
+  availability_zones = "${var.availability_zones}"
+  owner              = "${var.owner}"
+}
+
+module "jumpbox" {
+  source            = "../jumpbox"
+
+  cluster_name      = "${var.cluster}"
+  ssh_ip_address    = "${var.ssh_ip_address}"
+  vpc_id            = "${module.network.vpc_id}"
+  environment       = "${var.environment}"
+  enable_jumpbox    = "${var.enable_jumpbox}"
+  owner             = "${var.owner}"
+  key_name          = "${var.key_name}"
+  public_subnet_ids = "${module.network.public_subnet_ids}"
+}
+
 module "ecs_instances" {
-  source = "../ecs_instances"
+  source                  = "../ecs_instances"
```

Comment:
Terraform has a built in function https://www.terraform.io/docs/commands/fmt.html  that will manage spacing for you. Most editors should have a terraform plugin that will call this for you on save. 

The terraform standard is to only align '=' when there is a grouping of at least 2 lines.